### ISSUE TITLE
Identify learners for binary and continuous outcomes

### ISSUE THREAD

**Issue by ngreifer** [ecpolley/SuperLearner#142]
It would be nice if there was possibility of knowing which learners worked for binary vs. continuous outcomes. The only to find this out is to manually try them and receive an intimidating number of error messages if you guessed incorrectly. This could instead be stored as metadata for each learner and reported using `listWrappers()`, and if the user supplies an incompatible learner to `SL.library`, a single breaking error message could appear before running the full CV process.

**Comment by ecpolley** [comment]
Thanks for the suggestion. I think a table with some metadata on the included wrappers would be a nice addition, including
  * Type of outcomes
  * Allowed X variable input, matrix, data.frame, either, will a tibble break it?
  * Tuning parameters

Upfront checks are usually limited to only using the default wrappers, there is an internal function `.check.SL.library` that is called early in the code, but right now that only checks for the required package for the wrapper.

**Comment by ngreifer** [comment]
Yes, I think that would be helpful. I think the second bullet you mention might not be so valuable to users since ideally whatever kind of input the user gives is preprocessed before being passed to the fitting function.
